### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by kayjeee**
Hello Micro-verse changers to branch and code review suggestions

### CODE REVIEW

**Comment by kayjeee** [comment]
fixed the issues suggested I fix

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (Qoosim) [inline .github/workflows/linters.yml:56]: - [ ] Linters test do not run on this PR. Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set it up properly

File: README.md [hunk 2]
```diff
@@ -25,257 +19,80 @@ OPTIONAL SECTIONS:
...
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
+- [💻Getting started](#getting-Started)
   - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+- [Tech Stack](#tech-stack)
```
Comment (Qoosim) [inline README.md:30]: - [ ] Kindly follow this [link]() to make a professional README file. All the headings must be present except FAQ which is optional. You can simply put `N/A` under any heading that is not relevant to your project  at this moment.

File: README.md [hunk 3]
```diff
@@ -25,257 +19,80 @@ OPTIONAL SECTIONS:
...
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
```
Comment (Qoosim) [inline README.md:97]: - [ ] Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set up MIT license for your project.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @kayjeee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Fair README file. :heavy_check_mark: 




## Required Changes ♻️

- [x] The content of your `index.html` is supposed to be `Hello Microverse` not `Header, My Header`. Kindly fix that.

your-content

Project Requirements

project-rqt

- [x] It is required that your PR must have a descriptive summary that says what work you've done in the PR. At this moment, your PR has no descriptive summary. Kindly add descriptive summary to your PR.

pr-desc

- [x] Your PR also has conflict you need to resolve before your project can be approve. 

your-conflict



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (kayjeee) [reply .github/workflows/linters.yml:56]: I really struggle with this  issue
I can't get linters to work
any help would be appreciated 
should I start a new project on GitHub?

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @kayjeee , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that has falling linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @kayjeee , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green linters
✅ ... Great communication with the reviwer

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**